src/pages/quiz-page/Quiz.css

```diff
@@ -0,0 +1,57 @@
+
+.quiz-question-container{  
+    height: 60vh;
+    width: 50%;
+    display: flex;
+    margin: auto;
```

Comment:
Is there need to use margin: auto here? Because you are using display : flex here?